### DISCUSSION TITLE
What is the purpose of Slinger ?

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#411] [Q&A]
I know this makes me sound like a half-wit but I am a bit confused - let me explain.

I have a ProHD which I have just successfully unbricked using Slingplayer 1.5. It is connected to a Virginmedia TIVO here in the UK.
If I start up my Slingplay desktop 2.0.4.251 it streams the output successfully. The on-screen remote looks different from the way the TIVO remote used to look but I guess that's because a different one got downloaded into the Slingbox when I reconfigured it earlier. (I'll no doubt be having to ask how to fix this at some time )

So if I can do this already what is the point of the Sling Server ? Is it primarily to enable access from OUTSIDE of my LAN ?

**Comment by Toei79** [comment]
slingplay its not longer working at least for me.

**Comment by floormanager1** [comment]
You cannot use Slingplayer 2.0.4 for newer Slingboxes.  And if you want to use your Pro HD away from your home, then you need to allow internet traffic, which means it might be bricked again.  Also, the Iphone, Android and Firestick apps no longer work, so Slinger is the way.  I had planned to use my older Slingboxes with Slingplayer 2.0.4 configured with a direct connection after the 9th but Mr Sling decided to prevent me doing that my bricking them.  Cannot unbrick my Pro or Solo.

**Reply by GerryDazoo** [reply]
The boxes don't get bricked via an in coming port mapping. The boxes call home outgoing and then get bricked by a message sent on port 5678.  We're working on getting 5.x working so we can configure later boxes.

**Reply by Toei79** [reply]
> The boxes don't get bricked via an in coming port mapping. The boxes call home outgoing and then get bricked by a message sent on port 5678. We're working on getting 5.x working so we can configure later boxes.

Gerry but mrSling can find other ways to brick boxes? or just block that port make safe

i never thought someone comes with a solution so fast, i just bring back my slingboxes for the dust. 
more when we get same scenario for reciva radios , sadly those get bricked forever. after reciva dead. there was a half way solution.

**Reply by DoogieHowitzerWMD** [reply]
> I had planned to use my older Slingboxes with Slingplayer 2.0.4 configured with a direct connection after the 9th but Mr Sling decided to prevent me doing that my bricking them.

I hadn't realized that was possible with the older boxes.

Would that feasibly work on a LAN network that is without a WAN connection?

**Reply by floormanager1** [reply]
It was,  You could set it up on 2.0.0.4 as a direct connection.  Once set up, you could select it on 2.0.4

**Comment by gbjbasdw** [comment]
What worked for me...

Port blocked 5678
Unbricked my ProHD box using Slingplayer 1.5 (SlingPlayer-Setup-EU-1.5.1.343)
Use 2.0.4 to view the Slingbox on my LAN
BUT I have the wrong on-screen remote. It is using the Virgin V+ remote not the Virgin TIVO remote. Tried to fix but now I have NO remote at all :-(
